README.md

```diff
@@ -181,6 +181,7 @@ Contributors:
 * Eriksson Monteiro (UA.PT Bioinformatics)
 * Tiago Godinho (UA.PT Bioinformatics)
 * Jorge Miguel Silva (UA.PT Bioinformatics)
+* Rui Lebre (UA.PT Bioinformatics) - [@rlebre](https://github.com/rlebre)

```

Comment:
@rlebre Can we stay consistent here? All other contributors have a GitHub profile. We should either add them all, or keep that to maintainers only.